froide_fax/fax.py

```diff
@@ -58,6 +56,34 @@ def get_twilio_fax_data(fax_sid):
     return fax_data._properties
 
 
+def send_fax_telnyx(
```

Comment:
Maybe there should be a wrapper function `send_fax(to, media_url)` that wraps this function and adds the appropriate other arguments from settings.